lbmerchant93/petstrology
# Feature horoscope component

What does this PR do?
Create a dynamic route for the Horoscope page that brings you to the 'today' horoscope when you select a sign on the home page. Populates with the information from the API. If there is an error will display the error number. While loading, displays a loading message. 

What does it feature/fix?
Feature the today horoscope depending on the sign you select.

Where should the reviewer start?
Horoscope.js Line 1

How should this be tested?
From the home screen select a sign. Should be taken to a new page, url defines 'horoscope/(sign selected here)/today'. Should display today's horoscope information. 

What are the next steps or any issues/concerns?
Style the Horoscope component information
Create functionality for yesterday, today, and tomorrow
Test information displayed on Horoscope component page
